pom.xml

```diff
@@ -13,6 +13,11 @@
   <name>Jenkins jQuery plugin</name>
   <url>http://wiki.jenkins-ci.org/display/JENKINS/jQuery+Plugin</url>
 
+  <properties>
+    <jenkins.version>1.625</jenkins.version>
+    <java.level>6</java.level>
```

Comment:
There are arguments on both using a weekly or an LTS, so I'd say that both options are reasonable in many cases.

Anyway, for such an old baseline, I would pick the latest LTS in the line (the number of instances you may left behind should be fairly small) and you are requiring more fixes (in many lines including security ones). For a much newer baseline I would fine tune more to avoid forcing many people to upgrade.

In any case, for 1.625, java 7.